crates/wasi-preview1-component-adapter/src/lib.rs

```diff
@@ -108,9 +108,69 @@ pub mod bindings {
     });
 
     pub mod wit_bindgen_rt_shim {
+
         pub use bitflags;
 
         pub fn maybe_link_cabi_realloc() {}
+
+        // ===============================================================
+        // TODO: shouldn't have to copy this...
```

Comment:
I mentioned on the wit-bindgen PR but this is part of the "adding this type wreaks havoc" thing...